nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -0,0 +1,48 @@
+collision_monitor:
+  ros__parameters:
+    use_sim_time: True
+    base_frame_id: "base_footprint"
+    odom_frame_id: "odom"
+    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_out_topic: "cmd_vel"
+    transform_tolerance: 0.5
+    data_timeout: 5.0
+    stop_pub_timeout: 2.0
+    # Polygons represent safety area around the robot for "stop" and "slowdown" action types,
+    # and robot footprint for "circle" action type.
+    # Footprint could be "polygon" type with dynamically set footprint from footprint_topic
+    # or "circle" type with static footprint set by radius. "footprint_topic" parameter
+    # to be ignored in latest case.
+    polygons: ["PolygonStop"]
+    PolygonStop:
+      type: "polygon"
+      points: [0.5, 0.5, 0.5, -0.5, -0.1, -0.5, -0.1, 0.5, 0.5, 0.5]
+      action_type: "stop"
+      max_points: 3
+      visualize: True
+      polygon_topic: "polygon_stop"
+    PolygonSlow:
+      type: "polygon"
+      points: [0.5, 0.5, 0.5, -0.5, -0.1, -0.5, -0.1, 0.5, 0.5, 0.5]
+      action_type: "slowdown"
+      max_points: 3
+      slowdown_ratio: 0.3
+      visualize: True
+      polygon_topic: "polygon_slowdown"
+    FootprintApp:
+      type: "polygon"
+      action_type: "approach"
+      footprint_topic: "/local_costmap/published_footprint"
+      time_before_collision: 2.0
+      simulation_time_step: 0.02
+      max_points: 20
```

Comment:
Why so high for the approach case?

Reply:
Small number of max_points for approach will cause some oscillations due to scan sensors flickering:
```
[collision_monitor-2] [INFO] [1656053615.970321348] [collision_monitor]: Robot to approach for 2.000000 seconds away from collision
[collision_monitor-2] [INFO] [1656053616.059633501] [collision_monitor]: Robot to continue normal operation
[collision_monitor-2] [INFO] [1656053616.198632653] [collision_monitor]: Robot to approach for 2.000000 seconds away from collision
[collision_monitor-2] [INFO] [1656053618.303690934] [collision_monitor]: Robot to continue normal operation
[collision_monitor-2] [INFO] [1656053620.969416312] [collision_monitor]: Robot to approach for 2.000000 seconds away from collision
[collision_monitor-2] [INFO] [1656053631.682070997] [collision_monitor]: Robot to continue normal operation
[collision_monitor-2] [INFO] [1656053631.895262328] [collision_monitor]: Robot to approach for 2.000000 seconds away from collision
```
Good is that these changes are actually do not appear as "twitching" in simulation since we are using cmd_vel instead of odometry.
Higher values will cause robot to go more closely to the obstacle, which we should also avoid.
Changing the number down to `5`: it seems to be proper value from latest experiments.

Reply:
OK 5 is more reasonable :+1: